python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -517,3 +509,60 @@ def to_arrow(self):
                 arrow_polygons,
             ],
         )
+
+    def _align_to_index(
```

Comment:
Nice. To align to the index we only need to align type codes and offset buffers. That's a lot cheaper than realigning all the children. Can you add a brief comment to this function describing this?

Reply:
Done.